test/problems/second_quantization/vibrational/builders/test_vibrational_op_builder.py

```diff
@@ -9,7 +9,7 @@
 # Any modifications or derivative works of this code must retain this
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
-"""Tests Fermionic Operator builder."""
+"""Tests Vibrational Operator builder."""
```

Comment:
Do we need this in this PR? I know it is most probably from a copy-paste but maybe we do it in another PR for sanity? Either way is ok.

Reply:
Let's keep it - a very minor change.